### ISSUE TITLE
Hide album from request page 

### ISSUE THREAD

**Issue by pimseb** [AzuraCast/AzuraCast#3778]
Hello,
Is it a way to hide the album column on the song requests page? The page doesn't fit well on phones, that's why I'd like to show only artist / title and the request button.
Maybe editing the css could do this job? 
Thank you

**Comment by SlvrEagle23** [comment]
I've modified the Requests modal dialog/embed page to show a simplified "album art + title/artist" column as the default, but to add other columns as selectable in the same way as the media manager (via a dropdown at the top of the page), which should fix the issue of there being too many columns initially and allow selectability of extra columns if needed.

**Comment by pimseb** [comment]
Thank you for this update, it looks better now. However the covers are not shown on the request page. I only see the missing cover art for all songs.
